architectures/host/linux_driver/architecture/sof_driver_arch.rst

```diff
@@ -1,8 +1,18 @@
 .. _sof_driver_arch:
 
-SOF Driver Architecture
+SOF Linux Driver Architecture
 #######################
```

Comment:
I believe...
```
SOF Linux Driver Architecture
#######################
```
Needs to be 
```
SOF Linux Driver Architecture
#############################
```
Per [RST syntax](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings)